Finally pushing up initial article (#507)
Sage-Bionetworks/synapseDocs: articles/managing_teams_for_groups_and_projects.md

---
title: "Managing Teams for Groups and Projects"
layout: article
excerpt: Using Synapse teams for group permissions in projects. 
category: inpractice
---

# Creating Permissions Models for Synapse Projects

> Synapse supports several different kinds of project permissions. These are described in more detail as "sharing settings" in the article on [access controls](/articles/access_controls.html). Additionally, Synapse supports setting "local sharing settings" that govern permissions within a project; for example, if you want to make your project public while keeping some folders or data private, you would use local sharing settings to do this. Permissions and sharing settings within Synapse are otherwise hierarchical; that is, if you set permissions on a project, everything within that project inherits those permissions until local sharing settings have been created. 

Reviewer: ```suggestion
Synapse supports several different kinds of project permissions. These are described in more detail as "sharing settings" in the article on [access controls](/articles/access_controls.html#sharing-settings). Additionally, Synapse supports setting "local sharing settings" that allow you to make your project public while keeping some folders, files, or tables private. To do this, you would use local sharing settings. Permissions and sharing settings within Synapse are otherwise hierarchical; that is, if you set permissions on a project, everything within that project inherits those permissions until local sharing settings have been created. 
```

When thinking through the process of creating a permissions model for your project, consider the following questions:
* Who needs to be able to view this project? 
* Who needs to be able to edit or add content to this project?
* Who should be in charge of changing permissions on this project, to modify either of the above? 
* Does any content in this project need different permissions than the whole project? For example:

>   * Raw data vs. processed data folders with different permissions

Reviewer: ```suggestion
  * Raw vs. processed data folders with different permissions
```

Because permissions are additive, a user who is in all three teams will have the highest permissions granted; that is, if you add a single user to three different teams and grant those teams "admin", "edit", and "view" permissions, the user in question will have "admin" permissions. 

## Local Sharing Settings

Sometimes, users wish to retain private spaces within public projects, or otherwise create sharing settings on project components (folders, tables, etc.) that are different from the overall project permissions. This is possible using Local Sharing Settings. You can use teams for group permissions even with local sharing settings, which can help streamline administration even more. 

For example, to create a private folder within a public project, you would make the entire project public using the "Make Public" button. Then, create a new folder, and click on "Create Local Sharing Settings". Remove the two entries for the public, which are listed as follows:

> * All registered Synapse users - Can Download

Reviewer: ```suggestion
```

* Anyone on the web - Can View

The folder with then be shared only with the specific user groups that the entire project is shared with, and not the general public. Note that creating local sharing settings can sometimes alter the permissions model unintentionally; see below for how to triage your sharing settings. 

> # Triaging Project Permissions Using Views 

Reviewer: @kdaily can add in an example query.
